src/ol_infrastructure/applications/edxapp/Pulumi.applications.edxapp.mitxonline.CI.yaml

```diff
@@ -1,9 +1,29 @@
----
 secretsprovider: awskms://alias/infrastructure-secrets-ci
 encryptedkey: AQICAHjEYf3gV5ZTnviCJcHwGnUuLzld5EuYumgMrRde6bvXUgFX6tgmkn7GE+XzOn/yyBWiAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMC1y1VeTVSEM970yPAgEQgDvmhUTNKMDUFYzvNIhqWM/x9sCuXu1B4nxoOS9IBy8+UWgbC0JNOYpp3Vvzj9U9CuBFq5rpy4NR8qMpVg==
 config:
   aws:region: us-east-1
+  consul:address: https://consul-mitxonline-ci.odl.mit.edu
+  consul:http_auth:
+    secure: v1:5RkLPQCwui+wYVI4:VXZQblCIWeAGOHWUe1e5pkyQ2R0pcDx1FrSoCUwpYSavVpJMU+zPKKdUIagw7xHRFvrFFMlwaC2tnXLeuqb428LyrFtoF9Xm/pBRGyo=
+  consul:scheme: https
+  edxapp:business_unit: mitxonline
   edxapp:db_password:
-    secure: v1:IwTPN3Do+RpBiRsW:JpALSjS8dT3rByCB+/6Q9XK56GWu+eS6s+aKeEXa943g7ZbV5tsZ2iHisoyDJs3ESn8zdirxZuym9caIehTGcyvqfctVMz8IKJIiD7Up1AkX70rG8U0N2LHR
+    secure: v1:nTKkM6zA0wbJaq3T:LyxTy566zTW65zkQLLKsFGmCIPmPRB1rkXcsAg37hI+EUTX07XfX5TkpfdyBBRdntiLJ7odzNss=
+  edxapp:dns_zone: mitxonline
+  edxapp:docdb_cluster_id: tf-20210812221225580300000002
+  edxapp:domains:
+    lms: courses-ci.mitxonline.mit.edu
```

Comment:
Should this be `lms-ci` to keep the same patterns as QA/RC?

Reply:
We'll actually be changing to use `courses` for the other environments.